man/ocamlrun.m

```diff
@@ -69,22 +69,45 @@
 .B b
 flag in the OCAMLRUNPARAM environment variable (see below).
 .TP
+.B \-H
+Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+memory, and are not swapped.
+.TP
 .BI \-I \ dir
 Search the directory
 .I dir
 for dynamically-loaded libraries, in addition to the standard search path.
 .TP
+.B \-m
+Print the magic number of the bytecode executable given as argument and
+exit.
+.TP
+.B \-M
+Print the magic number expected for bytecode executable by this version
+of the runtime and exit.
+.TP
 .B \-p
 Print the names of the primitives known to this version of
 .BR ocamlrun (1)
 and exit.
 .TP
+.B \-t
+Increments debug trace level (ignored unless debug mode)
```

Comment:
Again, synchronise with the manual TeX file. The main sentence in the manual is better than here, but the bit in parenthesis is better here (if changed to "ignored unless **in** debug mode)

Reply:
I can't find any matching `\item["-t"] in manual/manual/cmds/runtime.etex`

Reply:
Hah - oops! I reviewed this with the diff for #9284 open side-by-side! I was referring to https://github.com/ocaml/ocaml/pull/9284/files#diff-961cc346f7a7a80522a2681bb26e465cR75-R76. So, on re-reading both, I'd suggest "Increments the trace level for the debug runtime (ignored by the other runtimes)"